packages/falter-berlin-ssid-changer/files/lib/lib_ssid-changer.sh

```diff
@@ -0,0 +1,26 @@
+#/bin/sh
+. /lib/functions.sh
+
+NODENAME=$(uci_get system @system[-1] hostname | cut -b -24 ) # cut nodename to not exceed 32 bytes
+OFFLINE_SSID="offline_""$NODENAME"
+
+check_connectivity() {
```

Comment:
This function should be changed to report the connectivity status as true / false and not an error count. All logic for the status should be in this function. After the first successful ping the function should terminate early and return true, performing the other pings is only necessary to determine the offline state.

Reply:
I considered that in the most recent version of the commit.